apps/docs/src/pages/admin/comments.tsx

```
@@ -369,7 +387,18 @@ export default function CommentsAdminDashboard(): React.ReactElement {
 
         {/* Review Modal */}
         {selectedComment && (
-          <div className={styles.modalOverlay} onClick={() => setSelectedComment(null)}>
+          <div
+            className={styles.modalOverlay}
+            onClick={() => setSelectedComment(null)}
+            onKeyDown={(e) => {
+              if (e.key === 'Escape') {
```

ESLint

[error] Replace `'Escape'` with `"Escape"`

(prettier/prettier)